addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxServerHandler.java

```diff
@@ -556,24 +560,66 @@ public void updateListener(SqueezeBoxPlayerEventListener listener) {
             }
         }
 
+        /**
+         * Returns a hexadecimal encoded MD5 hash for the input String.
+         *
+         * @param data
+         * @return
+         */
+        private String getMD5Hash(String data) {
+            String result = null;
+            try {
+                MessageDigest digest = MessageDigest.getInstance("MD5");
+                byte[] hash = digest.digest(data.getBytes("UTF-8"));
+                return bytesToHex(hash); // make it printable
+            } catch (Exception ex) {
```

Comment:
Do you really need to catch the Exception-class or are there more specific exceptions you could catch like the IOException. The current code might 'hide' programming errors like NullPointerExceptions occurring.

https://www.google.nl/search?q=java%20why%20not%20to%20catch%20exception